### PR TITLE
Install runtime core CSS for all AMP tests

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by cramforce** [comment]
This will make things so much better!!!!!!!!!!!!

File: testing/describes.js [hunk 1]
```diff
@@ -311,11 +318,29 @@ class RealWinFixture {
...
 
-        resolve();
+        // Intercept event listeners
+        interceptEventListeners(win);
+        interceptEventListeners(win.document);
+        interceptEventListeners(win.document.documentElement);
+        interceptEventListeners(win.document.body);
+        env.interceptEventListeners = interceptEventListeners;
+
+        if (!completePromise) {
```
Comment (jridgewell) [inline testing/describes.js]: You can do `resolve(completePromise)`, which will handle both cases.

**Comment by muxin** [comment]
https://github.com/ampproject/amphtml/pull/5629 include changes in this PR. Should we split it?

**Comment by dvoytenko** [comment]
@muxin Yes. Already rebased.

**Pull request merged**